meson.build

```diff
@@ -310,6 +325,12 @@ if get_option('enable-mpp')
           ''')
```

Comment:
Add the status of the new option to the list, line 301 of this file.

Reply:
change done